crates/re_viewer/src/app.rs

```diff
@@ -138,7 +137,10 @@ pub struct App {
     command_receiver: CommandReceiver,
     cmd_palette: re_ui::CommandPalette,
 
-    analytics: ViewerAnalytics,
+    // NOTE: Optional because it is possible to have the `analytics` feature flag enabled
+    // while at the same time opting-out of analytics at run-time.
+    #[cfg(feature = "analytics")]
+    analytics: Option<crate::viewer_analytics::ViewerAnalytics>,
```

Comment:
The old code encapsulated this `Option` and `#[cfg(feature = "analytics")]` so users of the analytics wouldn't need to care.

Now all uses of analytics will be littered with

```
#[cfg(feature = "analytics")]
if let Some(analytics) = &mut self.analytics {
    …
}
```

If we want to save CPU, we can still do

```
if self.analytics.is_enabled() {
    …
}
```

Reply:
It's not used a lot now, but maybe it will be?

Reply:
Yeah, it felt like a lot of `cfg`s where there should be just a few, but it makes more sense to move it back